app/models/deploy_service.rb

```diff
@@ -131,7 +131,9 @@ def notify_outbound_webhooks(deploy)
     deploy.stage.outbound_webhooks.each { |webhook| webhook.deliver(deploy) }
   end
 
-  def send_sse_deploy_update(type)
-    SseRailsEngine.send_event('deploys', type: type)
+  def send_deploy_update(finished = false)
+    count = Deploy.active_count
+    count -= 1 if finished # deploy is still active, so we substract one
```

Comment:
Very slight possibility of an off by one due to caching of `active_count`, but that's only if a deploy is shorter than 10 seconds :p

Reply:
yeah but eventually consistent ... previous code did a +1/-1 counter so it could have been off forever 🤷‍♀️